src/mlpack/core/data/dataset_mapper_impl.hpp

```diff
@@ -133,15 +130,14 @@ inline Datatype DatasetMapper<PolicyType>::Type(const size_t dimension) const
 template<typename PolicyType>
 inline Datatype& DatasetMapper<PolicyType>::Type(const size_t dimension)
 {
-  if (dimension >= types.size())
-    types.resize(dimension + 1, Datatype::numeric);
+  if (dimension >= types.size()) types.resize(dimension + 1, Datatype::numeric);
```

Comment:
I don't like one-line ifs, I think they are harder to read because they don't currently appear anywhere.  Can we change this back to the way it was?